bedrock/foundation/templates/foundation/annualreport/2020/index.html

```diff
@@ -0,0 +1,428 @@
+{% block page_image %}{{ static('img/foundation/annualreport/2019/share.jpg') }}{% endblock %}
+
+{% block body_id %}annual-2020{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('annual_report_2020') }}
+{% endblock %}
+
+{% block js %}
+  {{ js_bundle('annual_report_2020') }}
+{% endblock %}
+
+{% block content %}
+<main role="main">
+  <header class="c-hero-banner">
+    <div class="mzp-l-content">
+      <div class="c-banner-content">
+        <div class="c-header-container">
+          <h1>
+            <span class="left">
+                <strong class="l-highlight green">2020</strong><br>
+                <strong class="l-highlight green">-2021</strong>
+                <img src="{{ static('img/foundation/annualreport/2019/hero-dots.svg') }}" alt="">
+            </span>
+            <span class="right">State <br>of <br>Mozilla</span>
+          </h1>
+        </div>
+      </div>{#--/.c-banner-content--#}
+    </div>{#--/.mzp-l-content--#}
+  </header>
+
+
+  <div class="mzp-l-content mzp-t-content-lg c-page-intro">
+    <h2><span class="highlight dark">Our impact</span></h2>
+    <p>
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure pariatur sunt cum fuga aperiam adipisci numquam maiores fugiat eos, ipsa nam consequuntur. Delectus cupiditate unde, minima esse totam perspiciatis perferendis doloremque consectetur ipsa odio enim laboriosam accusamus quo voluptates temporibus nostrum assumenda fugit labore sed ratione voluptatum obcaecati consequuntur quae. Deleniti placeat dolor molestiae dolore nesciunt dolorem, pariatur deserunt! Aliquid provident porro veniam iure maiores? Optio deleniti temporibus accusantium nobis tempore aliquam ea vitae quod, accusamus quia iusto distinctio minus tempora nisi impedit error alias maxime dolor explicabo! Aperiam at temporibus, maxime itaque fuga incidunt molestiae est cumque? Et qui dicta impedit officia quasi explicabo assumenda. Suscipit, possimus. Ipsa eos quos voluptatem debitis laborum sequi, odio ea, dolorum ut voluptates dicta blanditiis corporis pariatur placeat voluptatum cumque exercitationem commodi, eius quisquam sit repellendus adipisci animi minus perspiciatis! Saepe sit nostrum, cum illo eaque fugit harum sequi, dolor debitis architecto aut.
+    </p>
+  </div>
+
+  <!-- Leadership Cards -->
+  <div class="mzp-l-content leadership no-js-hidden">
+    <h2 class="leadership-intro">A word<br> <strong>from our leadership</strong></h2>
+    <div class="mzp-l-card-third">
+
+      {{ card(
+        title='Mitchell Baker',
+        ga_title='Mitchell Baker',
+        tag_label='Steering Committee',
+        desc='Chief Executive Officer, Chairwoman of Mozilla Foundation',
+        class='has-modal',
+        attributes="data-modal-id=mitchell-baker-letter",
```

Comment:
attribute values should be in quotes, but here that means we get into a nested quote situation... this could be `attributes='data-modal-id="mitchell-baker-letter"'` but then the quotes get escaped, so it actually needs to be `attributes='data-modal-id="mitchell-baker-letter"|safe'`.

OR we could change it in the macro so the output is always safe like `{{ attributes|safe }}`